Geometry/TrackerCommonData/data/PhaseI/pixfwdCylinder.xml

```diff
@@ -61,6 +61,14 @@ root volume plus an anchor point for positioning.
  <Constant name="CylindersServiceZ0"   value="0*mm"/>
  <Constant name="CylindersServiceZMin" value="[CylindersServiceZoff]"/>
  <Constant name="CylindersServiceZMax" value="[CylindersOuterLength]"/>
+
+ <Constant name="CarbonFiberSheetNearIPRmin" value="101.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPRmax" value="168.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPThickness" value="0.2*mm"/>
+
+ <Constant name="FlexCableDepth" value="0.90*mm"/>
```

Comment:
@krose13 I still don't fully understand this, but I am happy to trust you here; I just wanted to make sure there is no typo involved.

If there are 3 0.3mm (real) cables in 0.9mm the density should be around that of the actual material? And I don't really understand how a material of mostly aluminium can have a 2-3x higher density than the metal: https://github.com/cms-sw/cmssw/pull/16316/files#diff-a6d3e039372043ee1df48846f91ee8b5R333